pom.xml

```
@@ -82,172 +99,140 @@
     </dependency>
     <dependency>
       <groupId>org.kie.server</groupId>
-      <artifactId>kie-server-common</artifactId>
+      <artifactId>kie-server-controller-client</artifactId>
     </dependency>
+
+    <!-- Quarkus -->
     <dependency>
-      <groupId>org.kie.server</groupId>
-      <artifactId>kie-server-api</artifactId>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy-jackson</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.kie.server</groupId>
-      <artifactId>kie-server-controller-client</artifactId>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-smallrye-health</artifactId>
     </dependency>
-
-    <!-- Thorntail Fractions -->
     <dependency>
-      <groupId>io.thorntail</groupId>
-      <artifactId>jaxrs</artifactId>
-      <exclusions>
-        <exclusion>
-          <artifactId>thorntail-jdk-specific</artifactId>
-          <groupId>io.thorntail.jdk-specific</groupId>
-        </exclusion>
-      </exclusions>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-config-yaml</artifactId>
     </dependency>
     <dependency>
-      <groupId>io.thorntail</groupId>
-      <artifactId>logging</artifactId>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-hibernate-orm-panache</artifactId>
     </dependency>
     <dependency>
-      <groupId>io.thorntail</groupId>
-      <artifactId>spi</artifactId>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-quartz</artifactId>
     </dependency>
     <dependency>
-      <groupId>io.thorntail</groupId>
-      <artifactId>microprofile-config</artifactId>
-      <exclusions>
-        <exclusion>
-          <artifactId>narayana-jts-idlj</artifactId>
-          <groupId>org.jboss.narayana.jts</groupId>
-        </exclusion>
-      </exclusions>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-flyway</artifactId>
```

if we're using flyway only for the tests, we should limit the scope of this artifact
```suggestion
      <artifactId>quarkus-flyway</artifactId>
      <scope>test</scope>      
```